ee/modules/160-multitenancy-manager/crds/projecttemplate.yaml

```
@@ -0,0 +1,142 @@
+
+                    The following values are also available in templates:
+                    - `{{ .projectName }}` — the name of the `Project` (environment) for which the template is being rendered.
+                    - `{{ .templateName }}` — the name of the `ProjectTemplate`.
+                    - `{{ .templateValues }}` — a dictionary of custom values, described in the [.spec.valuesSchema](cr.html#projecttemplate-v1alpha1-spec-valuesSchema) parameter and defined in the [.spec.template](cr.html#project-v1alpha1-spec-template) parameter.
+
+                    > **Note!** Specifying `.metadata.namespace` fields for objects is optional,
+                    > as they are automatically set with the name of the created environment (CR `Project`).
+                  type: string
+                subjects:
+                  description:
+                    List of ServiceAccounts, Groups and Users to provide
+                    access to the created environment (project).
+                  items:
+                    oneOf:
+                      - properties:
+                          kind:
+                            enum:
+                              - ServiceAccount
+                          name: {}
+                          namespace: {}
+                          role: {}
+                        required:
+                          - kind
+                          - name
+                          - namespace
+                          - role
+                      - properties:
+                          kind:
+                            enum:
+                              - User
+                              - Group
+                          name: {}
+                          role: {}
+                        required:
+                          - kind
+                          - name
+                          - role
+                    properties:
+                      kind:
+                        description:
+                          Kind of the target resource to apply access to the
+                          environment (`ServiceAccount`, `Group` or `User`).
+                        enum:
+                          - ServiceAccount
+                          - User
+                          - Group
+                        type: string
+                      name:
+                        description:
+                          The name of the target resource to apply access
+                          to the environment.
+                        minLength: 1
+                        type: string
+                      namespace:
+                        description:
+                          The namespace of the target resource to apply environment access to.
+
+                          Required only when using `ServiceAccount` from another namespace.
+                        pattern: "[a-z0-9]([-a-z0-9]*[a-z0-9])?"
+                        type: string
+                      role:
```

It's not a role, it's `accessLevel` from AuthorizationRule

```suggestion
                      accessLevel:
```